### PR TITLE
Add DAOs and bump version

### PR DESCRIPTION
**Pull request by ben-kaufman**
https://github.com/daostack/migration/blob/master/daos-params/migration-params-block-explode.json
https://github.com/daostack/migration/blob/master/daos-params/migration-params-pepper4d.json

### CODE REVIEW

File: ops/graph-deploy.js [hunk 1]
```diff
@@ -15,7 +15,7 @@ async function deploy (opts = {}) {
   let msg
 
   /* create the subgraph */
-  if (graphNode !== 'https://api.thegraph.com/deploy/') {
+  if (graphNode !== 'https://api.thegraph.com/deploy/' && graphNode !== 'https://api.staging.thegraph.com/deploy/') {
```
Comment (orenyodfat) [inline ops/graph-deploy.js]: does this works ?

File: daos/mainnet/pepper4d.json [hunk 2]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (orenyodfat) [inline daos/mainnet/pepper4d.json]: what is this daos ?

Comment (orenyodfat) [reply daos/mainnet/pepper4d.json]: please add a brief re this dao ? who asked it for ? who deploy it ?

File: daos/mainnet/blockexplode.json [hunk 3]
```diff
@@ -0,0 +1,9 @@
+{
```
Comment (orenyodfat) [inline daos/mainnet/blockexplode.json]: lease add a brief re this dao ? who asked it for ? who deploed it ?

File: ops/graph-deploy.js (hunk 1 above)
Comment (ben-kaufman) [reply ops/graph-deploy.js]: The deployment itself not yet (for me) but we should have this for when it will.

**Comment by orenyodfat** [comment]
does proposerRepReward =0 for pepper4d dao is per request ? 
does absolutemajority = 55% for block-explode is per request ?

**Comment by orenyodfat** [comment]
please rebase

**Pull request merged**